instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './CommentSection.css';
+
```

Comment:
An option is to make this stateful component to hold information for an input field.